bitnami/cassandra/README.md

```diff
@@ -72,6 +73,8 @@ The following tables lists the configurable parameters of the cassandra chart an
 | `cluster.rack`                           | Rack name                                                   | `rack1`                                                |
 | `cluster.enableRPC`                           | Enable Thrift RPC endpoint                                    | `true`                  |
 | `cluster.minimumAvailable`                           | Minimum nuber of instances that must be available in the cluster (used of PodDisruptionBudget)                                    | `1`                                                |
+| `cluster.internodeEncryption`                           | Set internode encryption. NOTE: A value different from 'none' requires setting `tlsEncryptionSecretName`                                   | `none`                                                |
```

Comment:
We usually use `nil` instead of `none`

Reply:
This is a cassandra value. You can set it to `none`, `all` and others. It is explained in the container documentation